type-definitions/dom-event.d.ts

```diff
@@ -0,0 +1,46 @@
+import {Stream} from "most";
+
+export function domEvent(event: string, node: EventTarget, capture: boolean): Stream<Event>
```

Comment:
I get the error `A parameter initializer is only allowed in a function or constructor implementation`
When trying to use a default value